doc/source/security/selinux.rst

```diff
@@ -0,0 +1,100 @@
+.. _selinux:
+
+===========================================
+Enabling SELinux 
+===========================================
+
+SELinux helps protects the user's home directory data. SELinux provides the following:
+
+* Prevents unauthorized users from exploiting the system
+
+* Allows authorized users to access files 
+
+* Used as a role-based access control system
+
+For more information, see `Percona Server and SELinux <https://www.percona.com/doc/percona-server/LATEST/security/selinux.html>`_
+
+Red Hat and CentOS distributes profiles to extend the SELinux policy module for mysqld. These profiles allow PXC-specific operations. They are:
+
+* percona-xtradb-cluster - extends the PS profile and allows the execution of SST scripts
+
+* wsrep-sst-xtrabackup-v2 - allows execution of the xtrabackup-v2 SST script
+
+Adjusting Policies
+------------------
+
+To adjust PXC-specific configurations, especially SST/IST ports, use the following procedures as ``root``:
+
+To enable port ``4567`` which allows group communication:
```

Comment:
Port 4567 is configured/enabled by default. The below procedure describes steps to configure port 14567 instead of the default 4567

Reply:
Rewritten